File: atividade04/python/login2.py
def login(user, password):
    valid_user = "admin"
    valid_pass = "1234"
    return user == valid_user and password == valid_pass
user = input("Usuario: ")
password = input("Senha: ")
if login(user, password):
    print("Acesso concedido")
else:
    print("Acesso negado")

File: atividade04/C/login2.c
#include <stdio.h>
#include <string.h>
int login(char user[], char pass[]) {
    char valid_user[] = "admin";
    char valid_pass[] = "1234";
    if (strcmp(user, valid_user) == 0 && strcmp(pass, valid_pass) == 0)
        return 1;
    return 0;
}
int main() {
    char user[50], pass[50];
    printf("Usuario: ");
    scanf("%s", user);
    printf("Senha: ");
    scanf("%s", pass);
    if (login(user, pass))
        printf("Acesso concedido\n");
    else
        printf("Acesso negado\n");
    return 0;
}